### ISSUE TITLE
Contribute to docs?

### ISSUE THREAD

**Issue by mikelove** [IGVF/catalog-front-end-issues#25]
Is there a way that IGVF teams can help contribute to documentation?

https://docs.catalog.igvf.org/introduction

**Comment by mikelove** [comment]
https://github.com/twlab/igvf-catalog-docs/tree/main

**Comment by mikelove** [comment]
E.g. https://github.com/twlab/igvf-catalog-docs/blob/main/nodes/disease.mdx

**Comment by hitz** [comment]
so I guess there are two ways to do this; give any editors push privs to this repo, or have Shane or someone edit a bunch of changes.    Otherwise I think we can close?

**Comment by mikelove** [comment]
Yes, closing. I think over the summer we (I can do this) should have the FG contribute new text (if they see a gap) and the easiest way would just be Google docs
